src/util/fees.cpp

```diff
@@ -26,15 +28,31 @@ std::string StringForFeeReason(FeeReason reason) {
     return reason_string->second;
 }
 
-bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
+const std::map<std::string, FeeEstimateMode>& FeeModeMap() {
     static const std::map<std::string, FeeEstimateMode> fee_modes = {
         {"UNSET", FeeEstimateMode::UNSET},
         {"ECONOMICAL", FeeEstimateMode::ECONOMICAL},
         {"CONSERVATIVE", FeeEstimateMode::CONSERVATIVE},
+        {(CURRENCY_UNIT + "/KB"), FeeEstimateMode::BTC_KB},
+        {(ToUpper(CURRENCY_ATOM) + "/B"), FeeEstimateMode::SAT_B},
     };
-    auto mode = fee_modes.find(mode_string);
+    return fee_modes;
+}
+
+std::string FeeModes(const std::string& delimiter = ", ") {
+    std::ostringstream r;
+    bool first = true;
+    for (const auto& i : FeeModeMap()) {
```

Comment:
This ends up with the uppercased strings in docs rather than the correct "BTC/kB" and "sat/B"

Reply:
Everything is uppercase right now. I'm honestly not sure if changing that is the right approach.

Reply:
Code has been updated to show correct capitalization.